syncopy/shared/errors.py

```diff
@@ -361,6 +369,30 @@ def SPYLog(msg, loglevel="INFO", caller=None):
     logfunc(PrintMsg.format(caller=" <" + caller + ">" if len(caller) else caller,
                           msg=msg))
 
+def log(msg, level="INFO", par=False, caller=None):
```

Comment:
So with the default log-level (`"IMPORTANT") this will emit nothing when just being called with: `spy.log("my message")?

Reply:
yes. I have changed the default log level of the function to IMPORTANT, so we need to do less typing.